Lexicon.Api/Repositories/CrudRepository.cs

```diff
@@ -20,7 +20,7 @@ public async Task<IEnumerable<T>> GetAllAsync()
 
     public async Task<T> GetAsync(object id)
     {
-        return await DbSet.FindAsync(id) ?? throw new InvalidOperationException($"{typeof(T).Name}Id {id} not found.");
```

Comment:
Why? :)

Reply:
It did not give an error if id gave a null result